example/completetest/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"os"
+	"strings"
+
+	"github.com/coder/serpent"
+	"github.com/coder/serpent/completion"
+)
+
+// InstallCommand returns a serpent command that helps
+// a user configure their shell to use serpent's completion.
+func InstallCommand() *serpent.Command {
```

Comment:
why export?

Reply:
Just forgot to change it from what you had initially